package.json

```diff
@@ -22,6 +22,7 @@
     "cloudfront:build:deploy:nc": "npm run client:build:deploy:nc && npm run cloudfront:invalidateCache",
     "cloudfront:update:build:deploy": "npm run cloudfront:setup && npm run cloudfront:build:deploy",
     "cloudfront:update:build:deploy:nc": "npm run cloudfront:setup && npm run cloudfront:build:deploy:nc",
+    "serverless:deploy": "serverless deploy",
```

Comment:
I think it should be removed because this is a duplication of line 18 ("cloudfront:setup": "sls deploy"). It is the same script.